packages/flutter/lib/src/services/platform_messages.dart

```diff
@@ -104,6 +103,8 @@ class PlatformMessages {
   ///
   /// Returns a [Future] which completes to the received response, decoded as a
   /// UTF-8 string, or to an error, if the decoding fails.
+  ///
+  /// Deprecated, use [PlatformMessageChannel.send] instead.
```

Comment:
Yes, that's the plan. Just wanted to land this PR first, then I'll remove uses of these methods.